docs/assets/javascripts/version-selector.js

```diff
@@ -0,0 +1,167 @@
+window.addEventListener("DOMContentLoaded", function() {
+    window.versionPages = {};
+    var VERSION = window.location.pathname.split("/")[1];
+    var VERSION_LATEST = ".";
+
+    function populateVersionSitemap(version) {
+        var versionPath = version === VERSION_LATEST ? "" : "/" + version;
+        window.versionPages[version] = [];
+
+        var xhrSitemap = new XMLHttpRequest();
+        var sitemapURL = window.location.origin + versionPath + '/sitemap.xml';
+        xhrSitemap.open("GET", sitemapURL);
+        xhrSitemap.onload = function() {
+            var xmlLoc = this.responseXML.getElementsByTagName("loc");
+            var nodeText = [];
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/codacy/docs/pullRequest?prid=26905)
```suggestion
        var sitemapURL = window.location.origin + versionPath + "/sitemap.xml";
```